cross/libbluray/Makefile

```diff
@@ -0,0 +1,19 @@
+PKG_NAME = libbluray
+PKG_VERS = 0.7.0
+PKG_EXT = tar.bz2
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = ftp://ftp.videolan.org/pub/videolan/libbluray/0.7.0
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/libxml2 cross/freetype cross/fontconfig
+
+HOMEPAGE =
```

Comment:
Maybe add some info for these entries here. Same for libopus and there might be others.